### ISSUE TITLE
How can I use a loss as the stopping criteria in Wasserstein GAN?

### ISSUE THREAD

**Issue by RamziFsm** [RamziFsm/WassersteinGAN#1]
Throughout the training step, the variation of generator loss and critic loss for 1000 epochs as following:
![a](https://user-images.githubusercontent.com/29406252/95318356-563abb80-088e-11eb-82f5-42bb7de4778a.jpg)
Does this variation look like correct?
How can I use a loss as the stopping criteria in Wasserstein GAN? Can I use Generator loss or critic loss? Can I use the early stopping method?
Thanks
